map_test.go

```diff
@@ -175,13 +176,17 @@ func testMap(t *testing.T, when spec.G, it spec.S) {
 const buildpackTOML = `
 [buildpack]
 id = "%[1]s"
-name = "%[1]s-name"
-version = "%[2]s"
+name = "%[2]s"
+version = "%[3]s"
 dir = "none"
 `
 
-func mkBuildpackTOML(t *testing.T, dir, id, version string) {
-	mkfile(t, fmt.Sprintf(buildpackTOML, id, version),
-		filepath.Join(dir, id, version, "buildpack.toml"),
+func mkBuildpackTOML(t *testing.T, dir, id, name, version string) {
+	mkfile(t, fmt.Sprintf(buildpackTOML, id, name, version),
+		filepath.Join(dir, escapeId(id), version, "buildpack.toml"),
 	)
 }
+
+func escapeId(id string) string {
```

Comment:
Should be `escapeID` as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

Also, the spec restricts the ID to `.`, `-`, and `/`, so a single underscore should be okay. (Less relevant in the tests,  just FYI.)